src/client/app/translations/data.ts

```diff
@@ -1390,7 +1399,7 @@ const LocaleTranslationData = {
 		"remove": "Eliminar",
 		"restore": "Restaurar",
 		"return.dashboard": "Regresar al panel principal",
-		"role": "Rol",
+		"role": "Role: ",
```

Comment:
I don't think this is the Spanish word for this. Maybe add the untranslated symbol.

Reply:
I'll fix this

Reply:
Completed